packages/app/obojobo-repository/shared/components/message-dialog.jsx

```diff
@@ -0,0 +1,19 @@
+require('./message-dialog.scss')
+
+const React = require('react')
+const Button = require('./button')
+
+const MessageDialog = props => {
+	return (
+		<div className="bulk-success-dialog">
```

Comment:
Nitpicking - change this class name from 'bulk-success-dialog' to 'message-dialog' to reflect the change to the component's name. Same change in the adjacent .scss file.